x/oracle/types/params.go

```diff
@@ -40,9 +39,6 @@ func validateDeposit(i interface{}) error {
 	if strings.TrimSpace(v) == "" {
 		return errors.New("deposit cannot be blank")
 	}
-	if _, err := sdk.AccAddressFromBech32(v); err != nil {
```

Comment:
Removing this check can put storage module on risk. It is better to have user error caught before spinning up the chain than processing buy and upgrade txs or other codes that uses it in the future.